dll/shellext/zipfldr/CZipCreator.cpp

```diff
@@ -315,6 +315,8 @@ unsigned CZipCreatorImpl::JustDoIt()
                                       Z_DEFAULT_STRATEGY,
                                       password,
                                       crc,
+                                      36,
```

Comment:
This needs a define somewhere.

Reply:
Added `MINIZIP_COMPATIBLE_VERSION` macro.